config/default.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
How about using development config as default?

Reply:
Someone may want to have prod config as default

Reply:
why?
In such case, they will run `build:prod` and will be sure that on local machine and travis they get the same result

Reply:
In case when env variable failed to be set, it's normal practice. As most popular config library on NPM does

Reply:
Please give some examples.
If you want to have default.config that it should be different from prod an dev. In another case, it doesn't make sense. If ENV is not provided that fallback with prod or dev

Reply:
I gave you an example when someone wants to have default as prod. They shouldn't care to change gulp task, they have to care of config's content, that's the normal flow

Reply:
But he is not able to run default config without modifying a package.json!
Furthermore, your comment below you have mentioned that things should be made obvious
![dev_prod environment config by sophieh29 pull request 372 gdg-x_hoverboard - google chrome 2018-01-11 14 47 15](https://user-images.githubusercontent.com/2954281/34826278-a33f554a-f6de-11e7-8ad8-b08e8f085d9d.png)

As I see a solution:
`build`: `development.json` - you will be able to test build with dev config
`build:prod`: `production.json` - used by travis.
`default.json` - should be deleted, or give links to projects that use this methodology.

Reply:
"But he is not able to run default config without modifying a package.json!"
He is able, by running just gulp

give links to projects that use this methodology.
https://github.com/lorenwest/node-config

^ this library, with half of million downloads in one month, is built on this methodology. In a case when there is not env set, a default will be taken. On my current project, we use it across all Node JS services.